internal/catalog/internal/controllers/endpoints/status.go

```diff
@@ -48,15 +45,17 @@ var (
 		Type:    StatusConditionBoundIdentities,
 		State:   pbresource.Condition_STATE_FALSE,
 		Reason:  StatusReasonNoWorkloadIdentitiesFound,
-		Message: identitiesNotFoundChangedMessage,
+		Message: "[]",
 	}
 )
 
 func ConditionIdentitiesFound(identities []string) *pbresource.Condition {
+	sort.Strings(identities)
+
 	return &pbresource.Condition{
 		Type:    StatusConditionBoundIdentities,
 		State:   pbresource.Condition_STATE_TRUE,
 		Reason:  StatusReasonWorkloadIdentitiesFound,
-		Message: fmt.Sprintf(identitiesFoundMessageFormat, strings.Join(identities, ",")),
+		Message: "[" + strings.Join(identities, ",") + "]",
```

Comment:
Did you consider just using `strings.Join`? `[` and `]` are making this quite a bit more complicated.

Reply:
Partly I did it for the humans reading it, but I'm also happy to remove them.

Reply:
I have a slight preference for removing the square brackets, but am fine with either.